pkg/annotations/annotations.go

```diff
@@ -192,6 +192,9 @@ const (
 	// VSMBNoDirectMap specifies that no direct mapping should be used for any VSMBs added to the UVM
 	VSMBNoDirectMap = "io.microsoft.virtualmachine.wcow.virtualSMB.nodirectmap"
 
+	// DisableWriteableFileShares disables adding any writable fileshares to the UVM
+	DisableWriteableFileShares = "io.microsoft.virtualmachine.fileshares.disablewritable"
```

Comment:
This is definitely a nit and not something we've been consistent about, but I prefer when we use the positive version of these settings for annotations. IOW "EnableWriteableFileShares" and "WCOWEnableGMSA". Thoughts @dcantah and @helsaawy?

Reply:
I picked disable to keep the default behavior/backwards compat of allowing writable shares unless explicitly disallowed
Having the annotation be `"EnableThis": "false"`, could also work, so I guess I have no preferences ...

Reply:
That makes sense. I could go either way in that case.